int32 gene_id
---
float64 distance